cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1874,13 +1875,12 @@ func mountNonDataPartitionMatchingKernelDisk(dir, fallbacklabel string, opts *sy
 	return doSystemdMount(partSrc, dir, opts)
 }
 
-func is24plusSystem(model *asserts.Model) bool {
-	switch model.Base() {
-	case "core20", "core22", "core22-desktop":
-		return false
-	default:
+func createSysrootMount() bool {
+	isCore24plus := osGetenv("CORE24_PLUS_INITRAMFS")
```

Comment:
I think we want a comment in code here, why we want to this based on initramfs and not the UC/hybrid version

Reply:
Good point, I've added a comment